azext_iot/central/params.py

```diff
@@ -214,13 +224,29 @@ def load_central_arguments(self, _):
 
     with self.argument_context("iot central device-group") as context:
         context.argument(
-            "api_version",
-            options_list=["--api-version", "--av"],
-            choices=CaseInsensitiveList(
-                [ApiVersion.preview.value, ApiVersion.v1_1_preview.value]
-            ),
-            default=ApiVersion.v1_1_preview.value,
-            help="The API version for the requested operation.",
+            "device_group_id",
+            options_list=["--device-group-id", "-d"],
+            help="Unique ID of the device group."
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name of the device group."
+        )
+        context.argument(
+            "filter",
+            options_list=["--filter"],
+            help="Query defining which devices should be in this group."
+        )
+        context.argument(
+            "description",
+            options_list=["--description"],
+            help="Short summary of device group."
+        )
+        context.argument(
+            "organizations",
+            options_list=["--organizations"],
+            help="List of organization IDs of the device group."
```

Comment:
use nargs for lists. By default everything is a string

Reply:
I see.